test/logic/ask_channel_reporting.test.ts

```
@@ -1,250 +1,329 @@
+  getValueFromProfile: jest.fn((profile: Profile, fieldId: string) => {
+    const value = profile?.fields?.[fieldId]?.value || "";
+    return value ? value.split(" ").splice(0, 3).join(" ") : "Unknown";
+  }),
+}));
+
+export interface MockedUser {
+  id: string;
+  team: string;
+  division: string;
+  department: string;
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)